usr/bin/ice

```diff
@@ -846,7 +851,7 @@ class Ice(Gtk.Window):
             sfile.write('user_pref("browser.cache.disk.smart_size.use_old_max"'
                         ', false);')
 
-            sfile.write('user_pref("browser.ctrlTab.previews", true);')
+            sfile.write('user_pref("browser.ctrlTab.previews", true);') 
```

Comment:
Whitespace? mb need to pep8 it all (with smth like Anaconda)

Reply:
I didn't catch that, I will fix it with my next pull request.